macros.in

```diff
@@ -942,7 +942,7 @@ package or when debugging this package.\
 %_sharedstatedir	%{_prefix}/com
 %_localstatedir		%{_prefix}/var
 %_lib			lib
-%_libdir		%{_exec_prefix}/%{_lib}
+%_libdir		@libdir@
```

Comment:
I think the reason it's like this is to support the weird multiarch schemes that some distributions use... 🤷‍♂️